torch/_tensor_docs.py

```diff
@@ -5937,13 +5937,29 @@ def callable(a, b) -> number
 add_docstr_all(
     "uniform_",
     r"""
-uniform_(from=0, to=1) -> Tensor
+uniform_(from, to, generator) -> Tensor
 
 Fills :attr:`self` tensor with numbers sampled from the continuous uniform
 distribution:
 
 .. math::
     P(x) = \dfrac{1}{\text{to} - \text{from}}
+
+Args:
+    from (float): lower-bound of the range to sample from
```

Comment:
To keep this PR simple, I'd suggest just removing this section for now as it doesn't seem to add new information like whether the bounds are inclusive are not. https://github.com/pytorch/pytorch/issues/16706